test/views/TopStories/test.index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import assert from 'assert';
+import sinon from 'sinon';
+import { shallow } from 'enzyme';
+import TopStories from '../../../views/TopStories/index.jsx';
+
+function selectDate(storiesWrapper) {
+  storiesWrapper.find('.sort-option').nodes[0].props.children[1].props.children.props.onChange({target: { value : 'date'}});
+}
+
+function selectRelevance(storiesWrapper) {
```

Comment:
you could just have 1 method called `selectOption` and pass in the wrapper + the value you want to select -> `selectOption(wrapper, 'date')`